components/raftstore/src/store/fsm/peer.rs

```diff
@@ -4118,6 +4118,8 @@ where
         source: metapb::Region,
     ) {
         self.register_split_region_check_tick();
+        self.fsm.peer.region_buckets = None;
```

Comment:
How about moving it to L4167? We may need to check all other places that `size_diff_hint` is reset.

Reply:
done.  Good point.